_posts/2015-11-23-my-first-couple-of-months-at-codurance.md

```diff
@@ -0,0 +1,71 @@
+---
+
+### Some background
+
+Some of you may know me from the various meetups in the city, especially my attendance at a number of [LJC](www.meetup.com/Londonjavacommunity/) and [LSCC](www.meetup.com/london-software-craftsmanship) meetup events. Attending these events I learnt about various conferences like **Devoxx**, **SoCraTes**, **JAX London**, **Java2Days**, **OpenFest**, and some others, and have had the good fortune of being at such events and even presenting at some of them. My subject was usually around [Adopt OpenJDK](https://java.net/projects/adoptopenjdk/pages/AdoptOpenJDK) and [Adopt-a-JSR](https://java.net/projects/adoptajsr/pages/Home) whilst wearing my [LJC](www.meetup.com/Londonjavacommunity/) Advocate hat, but I also took the chance to present on other topics.
+
+During this time I met a lot of people with various levels of experience and my interest and urge to learn more about the Java/JVM platform, Code Quality, Software Design, XP Practices, Software Craftsmanship, etc..., were on the rise and saw no end. And whilst attending these events I came across [Sandro](https://twitter.com/sandromancuso) and [Mash](https://twitter.com/mashooq), who were in those days hosting [LSCC](www.meetup.com/london-software-craftsmanship) events. I went to many of [LSCC](www.meetup.com/london-software-craftsmanship) events, especially liked the hands-on sessions (which are still my favourite). These events always took place in the evenings and sometimes during weekends, and in most instances organised at [Skillsmatter](https://skillsmatter.com/).
+
+I also noticed that many things I learnt at such events and conferences wouldn't always immediately recognised or accepted at the workplace. And moving to another work environment didn't always solve this problem fully. I found that I wasn't learning and applying what I wanted to from my peers and very few were really in tuned with what the community was about. So one fine day I decided to take charge of my career and make a serious decision and take up the Apprenticeship program offered by [Codurance](http://codurance.com/) and go through the process.
+
+I was urged to go this way after being inspired by [Sandro](https://twitter.com/sandromancuso)'s book: [The Software Craftsman](http://www.amazon.co.uk/books/dp/0134052501), attending all the [SoCraTes UK](http://socratesuk.org/) conferences, and meeting with developers who valued and took pride of their work namely their craft. 
+
+### Where we are just now
+
+It's now been nearly two months since I have been working for 'Codurance', a formidable force. And so its also about time that I share my experiences with my fellow mates and the community around me.
+
+During my first few weeks at [Codurance](http://codurance.com/), I have been busy learning various things that have been chalked out for becoming a craftsman.
```

Comment:
You should probably put the link to Codurance's website on the first mention of the company. No need to put it twice.